### PR TITLE
Utfcpp Project v2.3.5

### PR DESCRIPTION
**Pull request by stuertz**
An easily to handle generic library for UTF-8 encoded Unicode strings

### CODE REVIEW

**Comment by stuertz** [comment]
This package simply ships some headers, what kind of tests should be done?

**Comment by stuertz** [comment]
@conda-forge/staged-recipes : Hi, this is ready for merging, please have a look.

File: recipes/utfcpp/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,38 @@
+{% set name = "utfcpp" %}
+{% set version = "2.3.5" %}
+
+package:
+   name: {{ name|lower }}
+   version: {{ version }}
+
+source:
+   git_rev: {{ "v" ~ version }}
+   git_url: https://github.com/nemtrif/utfcpp.git
```
Comment (ocefpaf) [inline recipes/utfcpp/meta.yaml]: Please change this to a tarball instead.

Comment (stuertz) [inline recipes/utfcpp/meta.yaml]: Hi, thank you for the review. 
Yes I can do that, but please explain, why using a git tag is not as good as using a tarball. Esp. as long as git urls are supported natively by conda build. - thanks.

**Comment by stuertz** [comment]
Waiting for https://github.com/conda-forge/conda-smithy/pull/489 to fix the AppVeyor build

File: recipes/utfcpp/meta.yaml (hunk 1 above)
Comment (stuertz) [inline recipes/utfcpp/meta.yaml]: done.

**Comment by stuertz** [comment]
@conda-forge/staged-recipes I think this is ready for merge. Please have a kindly look. Chnages requested by @ocefpaf are done.

**Comment by stuertz** [comment]
Friendly nudge @ocefpaf ;-)

**Pull request merged**